tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+import os
+import pytest
+
+from wptserve import server
+
+
+@pytest.fixture
+def httpd():
+    HERE = os.path.dirname(os.path.abspath(__file__))
+    httpd = server.WebTestHttpd(port=8080,
+                                        doc_root=os.path.join(HERE, 'data'),
+                                        host='127.0.0.1')
+    return httpd
```

Comment:
Why did you remove `httpd.start()` and the yield? Both are still needed to first make the server available to the test, and to clean it up afterward.

Reply:
I won't be able to return an argument in the generator if there is a yield. How am I supposed to return httpd along with a yield httpd.start()?

Reply:
No, you have to yield `httpd` and not the return value from `httpd.start()` as I already mentioned in an earlier review comment. So it will look like the following:

```
httpd.start(block=False)
yield httpd
httpd.stop()
```

That will start the server, use `httpd` for the test, and once that has been finished stops the server.

Reply:
Please review the changes